src/Explorer/Controls/Settings/SettingsComponent.tsx

```diff
@@ -573,7 +573,7 @@ export class SettingsComponent extends React.Component<SettingsComponentProps, S
           defaultExperience: this.container.defaultExperience(),
           dataExplorerArea: Constants.Areas.Tab,
           tabTitle: this.props.settingsTab.tabTitle(),
-          error: reason.message
+          error: getErrorMessage(error, true)
```

Comment:
I agree with @srinathnarayanan's other comment. What about adding a new key to `traceFailure` that adds the stack?

`stack: getErrorStack(error)` can return undefined if it doesn't exist.

Reply:
Done